### DISCUSSION TITLE
Dynamic Forms

### DISCUSSION THREAD

**Discussion by jkergosi** [ivan-dalmet/formiz#104] [General]
I know you can see form fields with:

      const myForm = useForm();
      console.log(myForm.fields);

But can I actually  is FormIZ to create forms dynamically, providing the schema with JSON?  Ex. :

      const myForm = useForm();
      myForm.fields = fieldJson;

Where fieldJson has all required params to build the form.
{
  fieldA: {
    name: 'myFieldA'
    type: 'text'
    label: 'Field A'
    errorMessage: 'Field A is Required',
  },
  fieldB: {
    name: 'myFieldB'
    type: 'text'
    label: 'Field B'
    errorMessage: 'Field B is Required',
  },
...
}

**Comment by jkergosi** [comment]
Here is an example using react-hook-forks: 
https://dev.to/vineetpardhi/how-to-create-dynamic-forms-in-react-using-react-hook-forms-5gab

**Comment by ivan-dalmet** [comment]
Hello @jkergosi you can do the same with Formiz :)
If I have time I will write an example. (Just been a dad since few weeks)

**Reply by jkergosi** [reply]
Congrats Ivan!  

When you get a chance that would be awesome.  If you provide an example, I'd be glad to build out more advanced ones, which can be added to the documentation.

**Comment by ivan-dalmet** [comment]
@jkergosi I made a simple example here ➡️ https://codesandbox.io/s/formiz-dynamic-form-exx0r 🤗
